sphinx_rtd_theme/footer.html

```diff
@@ -8,44 +8,29 @@
         <a href="{{ prev.link|e }}" class="btn btn-neutral" title="{{ prev.title|striptags|e }}" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> {{ _('Previous') }}</a>
       {% endif %}
     </div>
-  {% endif %}
+  {%- endif %}
 
   <hr/>
 
   <div role="contentinfo">
     <p>
-    {%- if show_copyright %}
-      {%- if hasdoc('copyright') %}
-        {% trans path=pathto('copyright'), copyright=copyright|e %}&copy; <a href="{{ path }}">Copyright</a> {{ copyright }}.{% endtrans %}
-      {%- else %}
-        {% trans copyright=copyright|e %}&copy; Copyright {{ copyright }}.{% endtrans %}
-      {%- endif %}
-    {%- endif %}
-
     {%- if build_id and build_url %}
-      {% trans build_url=build_url, build_id=build_id %}
+      {%- trans build_url=build_url, build_id=build_id %}
         <span class="build">
           Build
           <a href="{{ build_url }}">{{ build_id }}</a>.
         </span>
-      {% endtrans %}
+      {%- endtrans %}
     {%- elif commit %}
-      {% trans commit=commit %}
+      {%- trans commit=commit %}
         <span class="commit">
           Revision <code>{{ commit }}</code>.
         </span>
-      {% endtrans %}
-    {%- elif last_updated %}
-      {% trans last_updated=last_updated|e %}Last updated on {{ last_updated }}.{% endtrans %}
+      {%- endtrans %}
     {%- endif %}
-
     </p>
   </div>
 
-  {%- if show_sphinx %}
-  {% trans %}Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>{% endtrans %}.
```

Comment:
@Blendify why was this removed? When building the demo docs, this does not show up anymore, so I don't think it is inserted elsewhere.

Reply:
I thought it was inherited from the main layout file but I think that is not working. I can add it back.